Docker-Swarm-deployment/single-node/nginx/example-configs/conf/rs.conf

```diff
@@ -0,0 +1,47 @@
+    # Main block for Resource server
+    server {
+        listen 443  ssl http2 ;
+        server_name         rs.io.test;
+
+	ssl_certificate     /etc/default-ssl/fullchain.pem;
```

Comment:
Add certs from letsencrypt -  /etc/nginx/certs/<server_name>/cert.pem;  and update the config